rotkehlchen/api/rest.py

```diff
@@ -1959,17 +1959,46 @@ def get_ignored_assets(self) -> Response:
             result = self.rotkehlchen.data.db.get_ignored_asset_ids(cursor)
         return api_response(_wrap_in_ok_result(list(result)), status_code=HTTPStatus.OK)
 
-    def add_ignored_assets(self, assets: list[Asset]) -> Response:
-        result, msg = self.rotkehlchen.data.add_ignored_assets(assets=assets)
+    def add_ignored_assets(self, assets_to_ignore: list[tuple[Asset, bool]]) -> Response:
```

Comment:
add a docstring here. It's no longer self-evident since there is a boolean argument.